src/challengePaths/2015/CommunityService.ts

```diff
@@ -60,11 +61,22 @@ const visitCouncil = () => visitUrl("council.php");
 const baseWeight = (): number =>
   have($effect`Fidoxene`) ? 20 : familiarWeight(myFamiliar());
 
+function hypotheticalModifier(
+  modifier: NumericModifier,
+  ...effects: Effect[]
+): number {
+  const newEffects = effects.filter((e) => !have(e));
+  return (
+    numericModifier(modifier) +
+    sum(newEffects, (effect) => numericModifier(effect, modifier))
+  );
+}
+
```

Comment:
This needs special-casing for any list of effects that includes Steely-eyed squint or Bow-legged swagger because:
```
numericModifier($effect`Steely-Eyed Squint`, "Item Drop")

Returned: 0.0

numericModifier($effect`Bow-legged swagger`, "Weapon Damage")

Returned: 0.0
```
This appears to always be so.

Reply:
Additionally, numericModifier results don't appear to be multiplied if things like that are active.